include/default-term.php

```diff
@@ -0,0 +1,226 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+/**
+ * Manages filters and actions related to default terms.
+ *
+ * @since 3.7
+ */
+class PLL_Default_Term {
+
+	/**
+	 * A reference to the PLL_Model instance.
+	 *
+	 * @var PLL_Model
+	 */
+	protected $model;
+
+	/**
+	 * Preferred language to assign to new contents.
+	 *
+	 * @var PLL_Language|null
+	 */
+	protected $curlang;
```

Comment:
What made you change `$ref_lang` for `$curlang`?
It's not the same thing in an admin context I think.

Reply:
But `$ref_lang` doesn't mean anything in non-admin context, so I wasn't sure which one to use. In doubt, I guess we should keep `$ref_lang` 🤔

Reply:
Sorry, I meant `$pref_lang`... 😅

Reply:
But I get what you say. Now I don't know what's better :/

Reply:
And since the current language is set to it, that's why I changed to `$curlang`.